test/salesforce/event-flow/main.go

```diff
@@ -0,0 +1,365 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/amp-labs/connectors"
+	"github.com/amp-labs/connectors/salesforce"
+	"github.com/amp-labs/connectors/test"
+	"golang.org/x/oauth2"
+)
+
+func main() {
+	creds, err := test.GetCreds("../../../creds.json")
```

Comment:
If you're going to hard-code paths like this, I'd add a comment on the top spelling out where you expect the program to be run from. Otherwise this will result in confusion.

Better would be to auto-discover the `.git` directory automatically and traverse from there -- but that's a fair amount of work, so a comment is probably fine.